docs/cheaha/slurm/practical_sbatch.md

```diff
@@ -192,7 +198,7 @@ sbatch --array=0-$FILE_COUNT job.sh
         Double parentheses with a leading dollar sign like `$((...))` are used for evaluating integer arithmetic to a variable.
     <!-- markdownlint-enable MD046 -->
 
-1. The line `sbatch --array=0-$FILE_COUNT job.sh` puts the array tasks in the Slurm queue using the `job.sh` script. The number of tasks runs from `0` to `$FILE_COUNT` as compute above.
+1. The line `sbatch --array=0-$FILE_COUNT%4 job.sh` puts the array tasks in the Slurm queue using the `job.sh` script. The array of tasks runs from 0 to $FILE_COUNT as determined earlier, with the %4 limit restricting the number of tasks that can run simultaneously to 4.
```

Comment:
The "limit restricting" is confusing. I see what you mean here but it took me a couple of reads. What about "where %4 limits the number of simultaneous tasks to 4."